server/src/test/java/org/opensearch/index/remote/RemoteSegmentTransferTrackerTests.java

```diff
@@ -152,15 +152,16 @@ public void testComputeTimeLagOnUpdate() throws InterruptedException {
         transferTracker.updateLocalRefreshTimeMs(currentTimeMsUsingSystemNanos());
 
         transferTracker.updateLatestLocalFileNameLengthMap(List.of("test"), k -> 1L);
-        // Sleep for 100ms and then the lag should be within 100ms +/- 20ms
-        Thread.sleep(100);
-        assertTrue(Math.abs(transferTracker.getTimeMsLag() - 100) <= 20);
+        // Sleep for 100ms and then the lag should not be shorter
+        long span = 100;
+        Thread.sleep(span);
+        assertTrue("Lag is not expected to be shorter than span [" + span + "ms]", transferTracker.getTimeMsLag() >= span);
```

Comment:
Primarily wanted to add transferTracker.getTimeMsLag(), which would help in debugging :)

Reply:
Good point. Fixed.